linchpin/FilterUtils/FilterUtils.py

```diff
@@ -0,0 +1,267 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import json
+from six.moves import range
+
+
+try:
+    import libvirt
+    from xml.dom import minidom
+    from xml.etree.ElementTree import fromstring
+except Exception as e:
+    print(e)
+
+
+def add_res_type(hosts, res_type):
```

Comment:
What's the purpose of moving all of these functions?  Are we unable to test them otherwise?

Reply:
Yes.
I tried importing them from their respective files but there was a path resolution issue. 
Further, Since most of the functions are utility based functions they can also be used in other places if consolidated in a sub package.